app/frontend/src/components/forms/manage/ExternalAPIs.vue

```diff
@@ -0,0 +1,603 @@
+<script>
+import { mapState, mapWritableState, mapActions } from 'pinia';
+import { useFormStore } from '~/store/form';
+import { formService } from '~/services';
+import { i18n } from '~/internationalization';
+import { useNotificationStore } from '~/store/notification';
+import { NotificationTypes } from '~/utils/constants';
+import BaseDialog from '~/components/base/BaseDialog.vue';
+import { ref } from 'vue';
+
+export default {
+  components: {
+    BaseDialog,
+  },
+  data() {
+    return {
+      loading: false,
+      headers: [
+        { title: 'Name', key: 'name' },
+        { title: 'URL', key: 'endpointUrl' },
+        { title: 'Status', key: 'code' },
+        { title: 'Actions', key: 'actions', align: 'end' },
+      ],
+      externalAPIAlgorithmList: [],
+      externalAPIStatusCodes: [],
+      items: [],
+      techdocsLink:
+        'https://developer.gov.bc.ca/docs/default/component/chefs-techdocs/Capabilities/Functionalities',
```

Comment:
Just so we don't forget, this link will need to be updated once the docs are in the https://github.com/bcgov/common-hosted-form-service-techdocs repo / Backstage.

Reply:
I've added a new ticket in the backlog to get tech docs published and this link updated.